README.md

```diff
@@ -3,49 +3,88 @@
 This project is about harvesting from external sources to a CKAN (local or remote) instance.  
 This project uses a custom Python library ([ckan-harvesters](https://pypi.org/project/ckan-harvesters/))in order to read and validate different types of data sources.  
 
-## data.json files
+Install locally
 
-[More info](harvester_ng/datajson/README.md)
+```
+python3 -m venv /path-to-harvester-env
+source /path-to-harvester-env/bin/activate
+pip install -r requirements.txt
+```
 
-## CSW
+## Run with CKAN 
 
-[More info](harvester_ng/csw/README.md)
+First copy `.env.example` to `.env` file
+
+```
+# add "ckan" (to use ckan:5000) into "webserver" container's hosts file
+export HOST_IP=`ip -4 addr show scope global dev docker0 | grep inet | awk '{print \$2}' | cut -d / -f 1`
```

Comment:
comment and code are not related.

Reply:
I think [this](https://github.com/datopian/ckan-ng-harvest/commit/9341f180716923dca91e8e3ddf9af00b5167358b) is better but not sure if I get what you say.  
We need this comand to run harvester & CKAN

Reply:
where is HOST_IP used? 
How is that command related to the comment above it?
On which systems does this work?

Reply:
I removed all references to HOST_IP. Since we update the containers name I think the problem I had is resolved.